src/orquestra/sdk/_base/_driver/_ce_runtime.py

```diff
@@ -465,12 +460,13 @@ def _list_wf_runs(
 
         for page_size in page_sizes:
             try:
-                # TODO(ORQSDK-684): driver client cannot do filtering via API yet
-                # https://zapatacomputing.atlassian.net/browse/ORQSDK-684?atlOrigin=eyJpIjoiYmNiZjUyMjZiNzg5NDI2YWJmNGU5NzAxZDI1MmJlNzEiLCJwIjoiaiJ9 # noqa: E501
+                # https://github.com/zapatacomputing/workflow-driver/blob/c52013c0f4df066159fc32ad38d489b3eaff5850/openapi/src/resources/workflow-runs.yaml#L14 # noqa: E501
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Usually we document these params closer to the API boundary, such as the `_models` module.

Reply:
[497750f](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/335/commits/497750f8df026839566c30b80c4611f5d447e936)